src/dnx.windows/dnx.windows.vcxproj

```diff
@@ -30,6 +30,9 @@
   <PropertyGroup>
     <IncludePath Condition="'$(RuntimeTargetOS)' == 'WIN7_PLUS_CORESYSTEM'">$(ProgramFiles)\Windows Kits\Win8CoreSystem\inc\crt;$(IncludePath)</IncludePath>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
```

Comment:
These vcxproj files are "carefully handcrafted artifacts" in order to keep the same settings for all native projects (and to remove duplication included by VS) - so you typically want to fix what VS adds.